Rubberduck.RibbonDispatcher/Sample Ribbons/SampleRibbon/src/MyResourceManager.cls

```diff
@@ -0,0 +1,109 @@
+VERSION 1.0 CLASS
+BEGIN
+  MultiUse = -1  'True
+END
+Attribute VB_Name = "MyResourceManager"
```

Comment:
`DefaultResourceManager`, or `ResourceManagerTemplate`, but not `MyResourceManager`, please.

Reply:
Agreed - just got stuck for a good name that day.